wiki/Contests/en.md

```diff
@@ -2,291 +2,414 @@
+| DJ TOTTO - Shoujo Alice to Hakoniwa Gensou Concerto | ![osu!][osu!] | ![][flag_CA] [J1\_](https://osu.ppy.sh/users/5918561) | ![][flag_CN] [yf\_bmp](https://osu.ppy.sh/users/4477199) | ![][flag_CN] [Lv9](https://osu.ppy.sh/users/7232130) | [Results](https://osu.ppy.sh/community/forums/topics/806794) |
+| Sota Fujimori - ACCELERATE | ![osu!][osu!] | ![][flag_HK] [Chaoslitz](https://osu.ppy.sh/users/3621552) | ![][flag_CN] [Koiyuki](https://osu.ppy.sh/users/2433507) | ![][flag_CN] [yf\_bmp](https://osu.ppy.sh/users/1243669) | [Results](https://osu.ppy.sh/community/forums/topics/806794) |
 
-- [Contest thread](https://osu.ppy.sh/community/forums/topics/620182)
-- [Results post](https://osu.ppy.sh/community/forums/topics/640906)
+#### [Pending Cup \#7 (2019)](https://osu.ppy.sh/community/forums/topics/953416)
 
-#### Pending Cup \#6 (2018)
+| Song | Mode | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |
+| :-- | :-: | :-- | :-- | :-- | :-- |
+| kamome sano - archive::zip | ![osu!][osu!] | ![][flag_FR] [Realazy](https://osu.ppy.sh/users/873961) | ![][flag_RU] [Senseabel](https://osu.ppy.sh/users/6184386) | ![][flag_US] [Axarious](https://osu.ppy.sh/users/2614511) | [Results](https://osu.ppy.sh/community/forums/topics/982516) |
+| EmoCo. - This Club is not 4 U | ![osu!][osu!] | ![][flag_CN] [Ellyu](https://osu.ppy.sh/users/4438154) | ![][flag_DE] [Lasse](https://osu.ppy.sh/users/896613) | ![][flag_RS] [Seolv](https://osu.ppy.sh/users/8067876) | [Results](https://osu.ppy.sh/community/forums/topics/982516) |
+| BEMANI Sound Team "Nekomata Master" - The last of world music | ![osu!][osu!] | ![][flag_PL] [Kalibe](https://osu.ppy.sh/users/3376777) | ![][flag_CA] [J1\_](https://osu.ppy.sh/users/5918561) | ![][flag_KR] [Heilia](https://osu.ppy.sh/users/9823042) | [Results](https://osu.ppy.sh/community/forums/topics/982516) |
 
-| Songs No. | Songs | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] |
-| :-- | :-- | :-- | :-- | :-- |
-| 1 | 米津玄师 - LOSER | [Skystar](https://osu.ppy.sh/users/873961) | [how2miss](https://osu.ppy.sh/users/4477199) | [AIR](https://osu.ppy.sh/users/2070688) |
-| 2 | DJ TOTTO - 少女アリスと箱庭幻想コンチェルト | [J1_](https://osu.ppy.sh/users/5918561) | [yf_bmp](https://osu.ppy.sh/users/1243669) | [Level9](https://osu.ppy.sh/users/7232130) |
-| 3 | Sota Fujimori - ACCELERATE | [Chaoslitz](https://osu.ppy.sh/users/3621552) | [Koiyuki](https://osu.ppy.sh/users/2433507) | [yf_bmp](https://osu.ppy.sh/users/1243669) |
+#### [Pending Cup \#8 (2020)](https://osu.ppy.sh/community/forums/topics/1145075)
 
-- [Contest thread](https://osu.ppy.sh/community/forums/topics/775847)
-- [Results post](https://osu.ppy.sh/community/forums/topics/806794)
+| Song | Mode | ![Gold Crown][GCrown] | ![Silver Crown][SCrown] | ![Bronze Crown][BCrown] | Results |
```

Comment:
despite all the winners using the same song, each of the category uses two songs. i'd probably change it to be like NPC 6